ci/container-image-build-task.yml

```diff
@@ -0,0 +1,28 @@
+# task that builds docker images with an option to specify a target platform.
+
+# After that command runs, you should have an `image/image.tar` oci image that you can test with:
+  # docker load -i image/image.tar
+  # docker tag LOADED_IMAGE_ID instana-agent-docker-s390x:local
+  # docker run --rm -it --entrypoint sh instana-agent-docker-s390x:local # this won't run if you are not on an s390x machine.
+
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: delivery.instana.io/int-docker-private-virtual/concourse/oci-build-task
+    tag: 0.11.0
```

Comment:
Should we use the latest here? Any reason for pinning the version?

Reply:
Not sure, I tend to pin versions, but latest should work just fine as well. Probably latest would not hurt here. :+1: